omnibus/config/templates/datadog-agent/systemd.sysprobe.service.erb

```diff
@@ -1,6 +1,7 @@
 [Unit]
 Description="Datadog System Probe"
-After=network.target
+Requires=sys-kernel-debug.mount
```

Comment:
were you able to confirm this `sys-kernel-debug` `mount` target would be available in general for all systemd distros?

Reply:
I can see it was commited in 2010, and supported CentOS/RHEL and debian/ubuntu versions came _after_ that, but it would be good to be sure :)